airflow/providers/salesforce/operators/bulk.py

```diff
@@ -47,7 +47,7 @@ class SalesforceBulkOperator(BaseOperator):
     def __init__(
         self,
         *,
-        operation: Literal[available_operations],
+        operation: Literal['insert', 'update', 'upsert', 'delete', 'hard_delete'],
```

Comment:
Is it not possible to do 
```suggestion
        operation: Literal[*available_operations],
```

Reply:
It’s a syntax limitation, Python does not allow using `*` in brackets. The previous `Literal[available_operations]` is valid Python [but PEP 586 considered this out of scope](https://peps.python.org/pep-0586/#rejected-or-out-of-scope-ideas).